pkg/agent/agent.go

```diff
@@ -216,14 +216,6 @@ func (k *Klocksmith) watchUpdateStatus(update func(s updateengine.Status)) {
 
 	go k.ue.ReceiveStatuses(ch, stop)
```

Comment:
This is being taken care of by #107 (at least this specific one). This function currently never stops cleanly so the defered close would never be called anyway.